qiskit/ignis/experiments/calibration/methods/basic_1q.py

```diff
@@ -87,53 +77,47 @@ class RoughAmplitudeCalibration(BaseCalibrationExperiment):
     """Performs a rough amplitude calibration by scanning the amplitude of the pulse."""
 
     def __init__(self,
-                 table: PulseParameterTable,
+                 inst_def: InstructionsDefinition,
                  qubit: int,
-                 data_processing: workflow.DataProcessingSteps,
-                 amp_vals: np.ndarray,
-                 analysis_class: Optional[Calibration1DAnalysis] = None,
-                 job: Optional = None,
-                 pulse_envelope: Optional[Callable] = None,
-                 pulse_name: Optional[str] = types.SingleQubitPulses.XP.value):
+                 data_processing: DataProcessingSteps,
+                 amp_vals: List,
+                 pulse_name: str,
+                 calibration_group: Optional[str] = 'default',
```

Comment:
why rough frequency cal doesn't have this argument?

Reply:
rough frequency will be a subsequent PR.